src/Murder/Core/Graphics/RenderContext.cs

```diff
@@ -371,10 +372,10 @@ public virtual void End()
         var scale = (_finalTarget.Bounds.Size.ToVector2() / _mainTarget.Bounds.Size.ToVector2());
         scale.Ceiling();
 
-        var cameraAdjust = (new Vector2(
-            Camera.Position.Point.X - Camera.Position.X - CAMERA_BLEED / 2,
-            Camera.Position.Point.Y - Camera.Position.Y - CAMERA_BLEED / 2) *
-            scale).Point;
+            var cameraAdjust = new Vector2(
```

Comment:
same: `    `

Reply:
(probably just format this file)

Reply:
Oh I think these were from sorting out a merge conflict where the competing change restyled the namespace like that changing the tabbing which I didn't pick up in the merge. You guys got all these changes? Sorry I was asleep haha